Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55e85eb677d3 <unknown>
#1 0x55e85e8c3688 <unknown>
#2 0x55e85e8e6b27 <unknown>
#3 0x55e85e8e268f <unknown>
#4 0x55e85e91d355 <unknown>
#5 0x55e85e9175a3 <unknown>
#6 0x55e85e8ecddc <unknown>
#7 0x55e85e8edde5 <unknown>
#8 0x55e85eb9849d <unknown>
#9 0x55e85ebb160c <unknown>
#10 0x55e85eb9a205 <unknown>
#11 0x55e85ebb1ee5 <unknown>
#12 0x55e85eb8e070 <unknown>
#13 0x55e85ebcd488 <unknown>
#14 0x55e85ebcd60c <unknown>
#15 0x55e85ebe6c6d <unknown>
#16 0x7f540127a609 <unknown>
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
====== WebDriver manager ======
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55d4772f27d3 <unknown>
#1 0x55d47704e688 <unknown>
#2 0x55d477071b27 <unknown>
#3 0x55d47706d68f <unknown>
#4 0x55d4770a8355 <unknown>
#5 0x55d4770a25a3 <unknown>
#6 0x55d477077ddc <unknown>
#7 0x55d477078de5 <unknown>
#8 0x55d47732349d <unknown>
#9 0x55d47733c60c <unknown>
#10 0x55d477325205 <unknown>
#11 0x55d47733cee5 <unknown>
#12 0x55d477319070 <unknown>
#13 0x55d477358488 <unknown>
#14 0x55d47735860c <unknown>
#15 0x55d477371c6d <unknown>
#16 0x7fa0de018609 <unknown>
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x56039bb287d3 <unknown>
#1 0x56039b884688 <unknown>
#2 0x56039b8a7b27 <unknown>
#3 0x56039b8a368f <unknown>
#4 0x56039b8de355 <unknown>
#5 0x56039b8d85a3 <unknown>
#6 0x56039b8adddc <unknown>
#7 0x56039b8aede5 <unknown>
#8 0x56039bb5949d <unknown>
#9 0x56039bb7260c <unknown>
#10 0x56039bb5b205 <unknown>
#11 0x56039bb72ee5 <unknown>
#12 0x56039bb4f070 <unknown>
#13 0x56039bb8e488 <unknown>
#14 0x56039bb8e60c <unknown>
#15 0x56039bba7c6d <unknown>
#16 0x7f5df5201609 <unknown>
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
====== WebDriver manager ======
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55a5a63f87d3 <unknown>
#1 0x55a5a6154688 <unknown>
#2 0x55a5a6177b27 <unknown>
#3 0x55a5a617368f <unknown>
#4 0x55a5a61ae355 <unknown>
#5 0x55a5a61a85a3 <unknown>
#6 0x55a5a617dddc <unknown>
#7 0x55a5a617ede5 <unknown>
#8 0x55a5a642949d <unknown>
#9 0x55a5a644260c <unknown>
#10 0x55a5a642b205 <unknown>
#11 0x55a5a6442ee5 <unknown>
#12 0x55a5a641f070 <unknown>
#13 0x55a5a645e488 <unknown>
#14 0x55a5a645e60c <unknown>
#15 0x55a5a6477c6d <unknown>
#16 0x7f3952379609 <unknown>
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55965ab4a7d3 <unknown>
#1 0x55965a8a6688 <unknown>
#2 0x55965a8c9b27 <unknown>
#3 0x55965a8c568f <unknown>
#4 0x55965a900355 <unknown>
#5 0x55965a8fa5a3 <unknown>
#6 0x55965a8cfddc <unknown>
#7 0x55965a8d0de5 <unknown>
#8 0x55965ab7b49d <unknown>
#9 0x55965ab9460c <unknown>
#10 0x55965ab7d205 <unknown>
#11 0x55965ab94ee5 <unknown>
#12 0x55965ab71070 <unknown>
#13 0x55965abb0488 <unknown>
#14 0x55965abb060c <unknown>
#15 0x55965abc9c6d <unknown>
#16 0x7f58ac9b3609 <unknown>
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
-199.4722456558482 -14.519752512110792
Score:  10
step time: 0.42399188299896196
Score:  10
step time: 0.3587809570017271
/home/alexzuzow/Desktop/agar_multiagent/utils.py:165: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  outputs.append(torch.nn.functional.softmax(classifier(digits[i]).squeeze(0)))
Score:  10
step time: 0.3752552649821155
Score:  10
step time: 0.37708823301363736
Score:  10
step time: 0.3541920720017515
Score:  10
step time: 0.3529025440220721
Score:  10
step time: 0.3522769410046749
Score:  10
step time: 0.44093612203141674

Score:  10
step time: 0.4494498370331712
Score:  10
step time: 0.41136966401245445
Score:  10
step time: 0.4271471580141224
Score:  10
step time: 0.43794095498742536
Score:  101
step time: 0.4261052010115236
Score:  10
step time: 0.4444072479964234
Score:  10
step time: 0.3851384340086952
Score:  10
step time: 0.38099302200134844
Score:  10
step time: 0.4337612820090726
Score:  10
step time: 0.5133480530348606
Score:  10
step time: 0.48649943998316303
Score:  11
step time: 0.4480081560323015
Score:  11
step time: 0.4449763799784705
Score:  11
step time: 0.47070470498874784
Score:  11
step time: 0.45592980697983876
Score:  11
step time: 0.46798497095005587
Score:  11
step time: 0.4715546100051142
Score:  11
step time: 0.4509657800081186
Score:  12
step time: 0.42199299595085904

Score:  12
step time: 0.4642982800141908
Score:  12
step time: 0.459913135971874
Score:  13
step time: 0.4485807860037312
Score:  13
step time: 0.46270439197542146
Score:  16
step time: 0.46329908299958333
Score:  16
step time: 0.48819374601589516
Score:  16
step time: 0.4538606330170296
Score:  16
step time: 0.4742530509829521
Score:  16
step time: 0.497152671974618
Score:  16
step time: 0.46137070102849975
Score:  16
step time: 0.5179270809749141
Score:  16
step time: 0.4911982900230214
Score:  16
step time: 0.5107489820220508
Score:  16
step time: 0.5351003810064867
Score:  16
step time: 0.5018691039877012
====== WebDriver manager ======
Current google-chrome version is 99.0.4844
Get LATEST chromedriver version for 99.0.4844 google-chrome
Driver [/home/alexzuzow/.wdm/drivers/chromedriver/linux64/99.0.4844.51/chromedriver] found in cache
Score:  0
step time: 0.6043565860018134
Score:  0
step time: 0.39212690299609676
Traceback (most recent call last):
  File "/home/alexzuzow/Desktop/agar_multiagent/train_DQN.py", line 124, in <module>
    state = agar1.reset()
  File "/home/alexzuzow/Desktop/agar_multiagent/agar.py", line 54, in reset
    self.driver = webdriver.Chrome(ChromeDriverManager().install(),options=self.chrome_options)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 70, in __init__
    super(WebDriver, self).__init__(DesiredCapabilities.CHROME['browserName'], "goog",
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py", line 93, in __init__
    RemoteWebDriver.__init__(
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 268, in __init__
    self.start_session(capabilities, browser_profile)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 359, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 422, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 421, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 443, in _request
    resp = self._conn.request(method, url, body=body, headers=headers)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/request.py", line 78, in request
    return self.request_encode_body(
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/http/client.py", line 1371, in getresponse
    response.begin()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/http/client.py", line 319, in begin
    version, status, reason = self._read_status()
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/http/client.py", line 280, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/home/alexzuzow/anaconda3/envs/myenv/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
KeyboardInterrupt